--- a/Modules/parameters.f90
+++ b/Modules/parameters.f90
@@ -19,7 +19,7 @@ MODULE parameters
   !! max number of different types of atom
   INTEGER, PARAMETER :: nsx = ntypx
   !! max number of atomic species (CP)
-  INTEGER, PARAMETER :: natx = 50
+  INTEGER, PARAMETER :: natx = 200 
   !! max number of atoms for DFT+U+V calculations
   INTEGER, PARAMETER :: sc_size = 1
   !! Defines the supercell in DFT+U+V as composed by the unit cells located
